scripts/getValidatorStartBlocks.js

```diff
@@ -0,0 +1,46 @@
+const path = require('path')
+require('dotenv').config({
+  path: path.join(__dirname, '../.env')
+})
+const Web3 = require('web3')
+const homeABI = require('../abis/HomeBridge.abi')
+const foreignABI = require('../abis/ForeignBridge.abi')
+const bridgeValidatorsABI = require('../abis/BridgeValidators.abi')
+
+async function getStartBlock(rpcUrl, bridgeAddress, bridgeAbi) {
+  try {
+    const web3Provider = new Web3.providers.HttpProvider(rpcUrl)
+    const web3Instance = new Web3(web3Provider)
+    const bridgeContract = new web3Instance.eth.Contract(bridgeAbi, bridgeAddress)
+
+    const deployedAtBlock = await bridgeContract.methods.deployedAtBlock().call()
+
+    const validatorContractAddress = await bridgeContract.methods.validatorContract().call()
+    const validatorContract = new web3Instance.eth.Contract(
+      bridgeValidatorsABI,
+      validatorContractAddress
+    )
+
+    const validatorAddedEvents = await validatorContract.getPastEvents('ValidatorAdded', {
+      fromBlock: 0,
```

Comment:
Maybe better use `deployedAtBlock` instead of `0`?

Reply:
Do you think it makes sense to extend functionality of Validator contract to have `deployedAtBlock` for this contract as well?

Reply:
@akolotov I think having `deployedAtBlock` on Validator contract will be useful for this case to avoid listening from block 0. If you agree with this, should I make a PR in the poa-bridge-contracts repo to add `deployedAtBlock` ?

Reply:
@akolotov Added a PR for this [https://github.com/poanetwork/poa-bridge-contracts/pull/54](https://github.com/poanetwork/poa-bridge-contracts/pull/54). Once it is merged, I can update this PR to use `deployedAtBlock` from BridgeValidators contract

Reply:
@patitonar I have approved https://github.com/poanetwork/poa-bridge-contracts/pull/54

Reply:
@patitonar merged